docs/ide.rst

```diff
@@ -722,7 +722,7 @@ Elpy allows to profile python scripts asynchronously using `cProfile`.
 
    Send the current buffer or region to the profiler and display the result with
    ``elpy-profile-visualizer``.
-   The default visualizer is `snakeviz`_, a browser-based graphical profile viewer that can be installed with `pip install snakeviz`.
+   The default visualizer is `snakeviz`_, a browser-based graphical profile viewer that can be installed with `pip install snakeviz`. RunSnakeRun (Python-2 only GTK GUI) and pyprof2calltree (uses QCacheGrind or KCacheGrind for display) are supported alternatives and can be similarly installed with pip.
```

Comment:
Could we add links for runsnakerun (http://www.vrplumber.com/programming/runsnakerun/)
and pyprof2calltree (https://github.com/pwaller/pyprof2calltree) ?

Reply:
Sorry for missing that.  Done! :-)  (with note about how pyprof2calltree seems to always be lowercase)